packages/react-scripts/config/jest/babelTransform.js

```diff
@@ -1,15 +1,18 @@
-// @remove-file-on-eject
+// @remove-on-eject-begin
 /**
  * Copyright (c) 2014-present, Facebook, Inc.
  *
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  */
+// @remove-on-eject-end
 'use strict';
 
 const babelJest = require('babel-jest');
 
 module.exports = babelJest.createTransformer({
   presets: [require.resolve('babel-preset-react-app')],
+  // @remove-on-eject-begin
   babelrc: false,
```

Comment:
Do we still create babelrc on eject? If we do I think it might be confusing that removing `babel-preset-react-app` from it won't actually remove it (because it's also inlined here and in webpack config).

Maybe we don't need to create babelrc on eject then?

Reply:
Yep, babel config still gets written to package.json on eject.  Yes, it could be confusing, and probably should not be there.  Will remove it.  Same thing for eslint config.

Reply:
FTR, removing babel config from package.json impacted mocha tests that run in e2e-kitchensink.  Babelrc was already being created in the e2e script itself, but only for tests before eject.  I opted to fix this by adding .babelrc to the fixture and removing the part of e2e script that was creating it.